### PR TITLE
add "how to create empty dictionaries" to the "The Dictionary Insight"

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/unordered-data-types/dictionary-standard-mapping-type.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/unordered-data-types/dictionary-standard-mapping-type.md?ref=4be07047a2297aede26ca2b7f1ecb14abd9c1e05)

**Review by Loopiezlol — DISMISSED** [review]
This looks great, but I would also add a mention in [this](https://app.enkipro.com/public/insight/5eb1862ba341dc051185ec0a) insight right at the end, showing that an empty dictionary could also be used to generate dicts from keys

File: python/python-core/unordered-data-types/dictionary-methods-ii.md [hunk 1]
```diff
@@ -73,10 +73,20 @@ surname = ('Hemsworth')
...
 
 print(famous_siblings)
-
-#{'Luke': 'Hemsworth', 'Chris': 'Hemsworth', 'Liam': 'Hemsworth'}
+# {'Luke': 'Hemsworth', 'Chris': 'Hemsworth', 'Liam': 'Hemsworth'}
 ```
 
+> 💡 Alongside `dict.fromkeys`, you can also use the alternative `{}.fromkeys` to generate a new dictionary with the given keys.
+
+If we update tha `famous_siblings` example, we would get the same result:
```
Comment (nem035) [inline python/python-core/unordered-data-types/dictionary-methods-ii.md:81]:
```suggestion
If we update the `famous_siblings` example, we would get the same result:
```

**Review by nem035 — APPROVED** [review]

File: python/python-core/unordered-data-types/dictionary-standard-mapping-type.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ revisionQuestion:
 
 The **dictionary**, `dict`, is Python's main *mapping type*. Mapping objects are ones that contain *key-value pairs*. 
 
+Here's how you can create an empty dicitonary:
```
Comment (kapnobatai137) [inline python/python-core/unordered-data-types/dictionary-standard-mapping-type.md:31]:
```suggestion
Here's how you can create an empty dictionary:
```

File: python/python-core/unordered-data-types/dictionary-standard-mapping-type.md [hunk 3]
```diff
@@ -28,6 +28,15 @@ revisionQuestion:
...
+```python
+usingBrackets = {}
+
+usingDict = dict()
+```
+
+Here is a sample of a pupulated dictionary:
```
Comment (kapnobatai137) [inline python/python-core/unordered-data-types/dictionary-standard-mapping-type.md:38]:
```suggestion
Here is an example of a populated dictionary:
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
@Stefan-Stojanovic looks good! we need to fix two typos and then it's safe to merge

**Review by kapnobatai137 — APPROVED** [review]
@Stefan-Stojanovic nice, let's get this live :rocket:

**Pull request merged**